gcs/gcs-generic.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+ name: gcs-generic
+spec:
+  workspaces:
+  - name: credentials
+    description: A secret with a service account key to use as GOOGLE_APPLICATION_CREDENTIALS.
+  params:
+  - name: command
+    description: The command line that you would like to enter.
+    type: string 
+  - name: subcommand
+    description: The subcommand line that you would like to enter.
+    type: array 
+  - name: serviceAccountPath
+    description: |
+      The path inside the credentials workspace to the GOOGLE_APPLICATION_CREDENTIALS key file.
+    type: string
+    default: service_account.json
+  steps:
+  - name: run-gsutil
```

Comment:
This doesn't actual run the gsutil command. We should either rename this or combine this.

Leaning towards combining since idk if there's added value in splitting this up into 2 containers if they both use the same image.

Reply:
there isn't a good way to combine unfortunately. We need script mode here for the auth activate. And we need command and args fields since array params are expanded if used within a [larger string array](https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#substituting-array-parameters)

Reply:
It will get error if combine these two, so we will have to split it into two contianers.